src/components/alert-box/alert-box.tsx

```diff
@@ -0,0 +1,96 @@
+import { useTheme, Button } from 'native-base';
+import React from 'react';
+import { Modal, View, Text } from 'react-native';
+
+import { AlertType } from '../../types/alert';
+
+import { AlertStyles } from './alert.styles';
+
+interface Props {
+  confirmText: string;
+  message: string;
+  onClose: () => void;
+  type: AlertType;
+  title: string;
+}
+
+const SYMBOL = {
+  [AlertType.DANGER]: '\u2718',
+  [AlertType.INFO]: '\u24D8',
+  [AlertType.SUCCESS]: '\u2714',
+  [AlertType.WARNING]: '\u26A0',
+};
+
+export const AlertBox: React.FC<Props> = ({ type, title, message, onClose, confirmText }) => {
+  const { colors } = useTheme();
+  const styles = AlertStyles();
```

Comment:
Can we follow the pattern for styling used in `Button` component [here](https://github.com/jalantechnologies/react-native-template/blob/main/src/components/button/button.tsx) for consistency?

In general, it is best to align the code pattern with existing code base so code remains consistent and hence easy to work with, unless you *strongly* disagree with that pattern.